kerastuner/tuners/hyperband.py

```diff
@@ -11,29 +11,43 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import copy
+import math
 import random
 
 from ..engine import multi_execution_tuner
 from ..engine import oracle as oracle_module
-from ..engine import trial as trial_lib
 
 
 class HyperbandOracle(oracle_module.Oracle):
     """Oracle class for Hyperband.
 
+    Note that to use this Oracle with your own subclassed Tuner, your Tuner
```

Comment:
"must understand three special hyperparameters" should be defined more precisely (in terms of specific instruction for the implementer)

Reply:
Done, added more context and also added an example of how `run_trial` should use